packages/flame/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
  - Adding `SpriteAnimationGroupComponent`
  - Allow isometric tile maps with custom heights
  - Add a new renderRect method to Sprite
+ - Improve camera code
```

Comment:
I think we could have a more specific entry here, if the changes are too big to fit a single entry you could add multiple entries